devices/Dhtxx/README.md

```diff
@@ -1,6 +1,7 @@
 # DHTxx - Digital-Output Relative Humidity & Temperature Sensor Module
 
-**IMPORTANT** This ensor is very time sensitive. This implementation will only work on few boards. It doesn't work on nay ESP32. Don't use this implementation for those MCU.
+> **IMPORTANT** This sensor is very time sensitive. This implementation will only work on few boards. It doesn't work on nay ESP32. Don't use this implementation for those MCU.
```

Comment:
A bit confusing the comment... it's not clear if it works or not with ESP32. Suppose it's better to just tell people to use the ESP32 version when using that platform... 

Anyway: typo in ANY above.